### DISCUSSION TITLE
Enable HTTPS using certificate crt and key file

### DISCUSSION THREAD

**Discussion by rashmimaharana** [drakkan/sftpgo#715] [Q&A]
Hi @drakkan 

I have both certifcate and key file. I would like to enable https and enable port 443. Can you please help me with config changes that required. I tried with below changes only, didn't work. Looks like i am missing something.

httpd->bindings->enable_https = true
httpd->bindings->port = 443
httpd-> certificate_file = "/tmp/domain.crt"
httpd-> certificate_file = "/tmp/domain.key"

Getting this error
2022-02-09T13:47:52.383 WRN error loading configuration file: While parsing config: invalid character 'U' in string escape code

**Comment by drakkan** [comment]
Hi,

your configuration file seems invalid, please post it

**Reply by rashmimaharana** [reply]
Thanks @drakkan 
We managed to enable HTTPS from GKE ingress level.
